project/backend/doc-server/src/main/resources/files/dd398660-ca95-4b58-97c8-5c7c5d2a8a31.py

```
@@ -0,0 +1,128 @@
+import numpy as np
+from nltk.tokenize import sent_tokenize
+from transformers import BertTokenizer, BertModel
+from flask import Flask, request, jsonify
+from dotenv import load_dotenv
+from flask_cors import CORS
+import requests
+os.environ["KMP_DUPLICATE_LIB_OK"] = "TRUE"
+
+load_dotenv()
+
+api_key = os.getenv('gemini_api_key')
```

Ruff (0.8.2)

Use capitalized environment variable `GEMINI_API_KEY` instead of `gemini_api_key`

(SIM112)